akka-javasdk/src/main/resources/reference.conf

```diff
@@ -92,4 +92,6 @@ akka.javasdk {
       collector-endpoint = ${?COLLECTOR_ENDPOINT}
     }
   }
+  # The comma separated list of FQCNs of components to exclude from running
+  components.exclude = ""
```

Comment:
Should it be `exclude` or `disable`? 
Both are good of course. I'm just trying to imagine if one is easier to understand then the other. 

If we say `exclude` will people ask: exclude from what? 
I don't know.

Reply:
`disable` or `disabled` ?

Reply:
`disable` as in imperative tense. Like in `exclude`.

Reply:
changed